SKILL.tsv

```diff
@@ -4077,7 +4077,7 @@ SKILL_20150428_004076	{memo X}{#DD5500}{ol}[Physical] - [Pierce]{/}{/}{nl}Attack
 SKILL_20150428_004077	{memo X}Attack per strike #{SkillAtkAdd}#{nl}Use #{CaptionRatio}# SP per 0.5 seconds
 SKILL_20150428_004078	$Maximum Duration: #{CaptionTime}# seconds{nl}Able to defend from ranged projectiles and magic
 SKILL_20150428_004079	$Maximum Purchase Quantity: #{CaptionRatio}#
-SKILL_20150428_004080	Remove magic circle nearby{nl}Max Duration 30 seconds{nl}Target #{CaptionRatio}#
+SKILL_20150428_004080	Remove magic circle nearby{nl}Max Duration: 30 seconds{nl}Target: #{CaptionRatio}#
```

Comment:
Remove magic circle nearby{nl}Max Duration: 30 seconds{nl}Target: #{CaptionRatio}#

to

Removes nearby Magic Circles{nl}Max Duration: 30 seconds{nl}Targets: #{CaptionRatio}#